bpf/lbmap/ipv4.go

```diff
@@ -38,6 +38,11 @@ var (
 		int(unsafe.Sizeof(RevNat4Key{})),
 		int(unsafe.Sizeof(RevNat4Value{})),
 		maxEntries)
+	RRSeq4Map = bpf.NewMap("cilium_lb4_rr_seq",
+		bpf.MapTypeHash,
+		int(unsafe.Sizeof(Service4Key{})),
+		int(unsafe.Sizeof(ServiceRRSeq{})),
```

Comment:
I can change that name. Thinking of it isn't is more obvious / readable looking at these two maps that they have same key. Why would you want to have a type RRSeq4Key Service4Key and hide this ?

        Service4Map = bpf.NewMap("cilium_lb4_services",
                bpf.MapTypeHash,
                int(unsafe.Sizeof(Service4Key{})),
                int(unsafe.Sizeof(Service4Value{})),
                maxEntries)
        RRSeq4Map = bpf.NewMap("cilium_lb4_rr_seq",
                bpf.MapTypeHash,
                int(unsafe.Sizeof(Service4Key{})),
                int(unsafe.Sizeof(ServiceRRSeq{})),
                maxFrontEnds)